documentation/asciidoc/accessories/camera/libcamera_apps_getting_help.adoc

```diff
@@ -0,0 +1,19 @@
+=== Getting Help
+
+For further help with `libcamera` and the `libcamera-apps`, the first port of call will usually be the https://www.raspberrypi.org/forums/viewforum.php?f=43[Rasperry Pi Camera Forum]. Before posting, it's helpful to:
+
+* Ensure your xref:../computers/os.adoc#using-apt[software is up to date].
+
+* If you are using _Buster_  or an earlier version of Raspberry Pi OS where you have had to build `libcamera` and `libcamera-apps` for yourself, or indeed if you are using a later version but have built them for yourself anyway, then you may need to update those _git_ repositories and repeat the build process.
+
+* Make a note of your operating system version (`uname -a`).
```

Comment:
Does this contain enough info to be able to tell if the user is using Buster or Bullseye, or is that extra info that the user should provide too?

Reply:
I think `uname -a` gives us that